core/src/main/java/nu/yona/server/exceptions/InvalidDataException.java

```diff
@@ -85,4 +85,9 @@ public static InvalidDataException excessEmailAddress()
 	{
 		return new InvalidDataException("error.user.email.address.not.supported");
 	}
+
+	public static InvalidDataException excessGoals()

```

Comment:
The scenario really is that e-mail addresses are not supported here. Excess means "surpassing usual, proper, or specified limits". In this case, the limit is "zero". In that case, it's more clear to say "goals not supported", as the message ID says.